### PR TITLE
Feat add api

### PR DESCRIPTION
**Pull request by LePetitTim**

### CODE REVIEW

File: georiviere/api/valorization/serializers/map.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            prefix = f'{obj.portal.pk}/'
+        if len(layer_type) == 2:
+            filter_type = layer_type[-1]
+            return f'{prefix}{layer_type[0]}/category/{filter_type}.geojson'
+        return f'{prefix}{layer_type[0]}.geojson'
+
+    def get_url(self, obj):
+        layer_type = obj.layer_type.split('-')
+        if layer_type[0] in ['pois', 'streams']:
+            return f'{obj.portal.pk}/{layer_type[0]}'
```
Comment (submarcos) [inline georiviere/api/valorization/serializers/map.py:43]: idem

File: georiviere/api/valorization/serializers/portal.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        return obj.spatial_extent.extent
+
+    def to_representation(self, instance):
+        ret = super().to_representation(instance)
+        ret['group'].append(
+            OrderedDict({'label': None,
+                         'layers': MapLayerSerializer(
+                             instance.layers.filter(group_layer__isnull=True),
+                             many=True,
+                         ).data}))
```
Comment (submarcos) [inline georiviere/api/valorization/serializers/portal.py:29]: only if there us layer without groups

File: georiviere/api/valorization/serializers/zoning.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+class WatershedTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = WatershedType
+        fields = ("name", "color")
+
+
+class WatershedSerializer(GeoFeatureModelSerializer):
+    type = WatershedTypeSerializer(many=False, source='watershed_type')
```
Comment (submarcos) [inline georiviere/api/valorization/serializers/zoning.py:36]:
```suggestion
    type = WatershedTypeSerializer(source='watershed_type')
```

File: georiviere/api/valorization/serializers/portal.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    spatial_extent = SerializerMethodField()
+
+    class Meta:
+        model = Portal
+        fields = (
+            'id', 'name', 'map', 'group', 'spatial_extent'
+        )
+
+    def get_spatial_extent(self, obj):
+        return obj.spatial_extent.extent
```
Comment (submarcos) [inline georiviere/api/valorization/serializers/portal.py:20]: exception if no spatial extent defined

File: georiviere/portal/models.py [hunk 5]
```diff
@@ -1,20 +1,88 @@
...
                              default='')
-    description = models.TextField(verbose_name=_("Description"), help_text=_("Description on Georiviere"),
+    description = models.TextField(verbose_name=_("Description"),
+                                   help_text=_("Description on Georiviere website used for Search Engine Optimization"),
                                    default='')
     main_color = ColorField(verbose_name=_("Main color"), default='#444444',
                             help_text=_("Main color"))
+    min_zoom = models.PositiveSmallIntegerField(default=0)
+    max_zoom = models.PositiveSmallIntegerField(default=22)
+    spatial_extent = PolygonField(srid=settings.SRID, null=True)
```
Comment (submarcos) [inline georiviere/portal/models.py:85]: default value with admin spatial_extent ?

File: georiviere/api/valorization/serializers/portal.py (hunk 4 above)
Comment (LePetitTim) [reply georiviere/api/valorization/serializers/portal.py]: If there is no spatial extent i give spatial extent of the project 
(settings.Spatial_extent)

Comment (submarcos) [reply georiviere/api/valorization/serializers/portal.py]: this is not the case

File: georiviere/portal/admin.py [hunk 6]
```diff
@@ -1,12 +1,127 @@
...
+    ordering_field = "order"
+    list_editable = ["order"]
+    list_display = ('label', 'url', 'order')
+    search_fields = ('label', )
+
+
+class MapGroupLayerAdminForm(forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        super(MapGroupLayerAdminForm, self).__init__(*args, **kwargs)
```
Comment (submarcos) [inline georiviere/portal/admin.py:78]:
```suggestion
        super().__init__(*args, **kwargs)
```

File: georiviere/portal/locale/en/LC_MESSAGES/djangojs.po [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: static/river/js/remove_delete_button.js:6
```
Comment (submarcos) [inline georiviere/portal/locale/en/LC_MESSAGES/djangojs.po:21]: --no-location ?

File: georiviere/portal/locale/fr/LC_MESSAGES/djangojs.po [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+"PO-Revision-Date: 2021-11-23 15:51+0000\n"
+"Last-Translator: Jean-Etienne Castagnede <jecastagnede@vivaldi.net>, 2021\n"
+"Language-Team: French (https://www.transifex.com/georiviere/teams/128005/fr/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: fr\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: static/river/js/remove_delete_button.js:6
```
Comment (submarcos) [inline georiviere/portal/locale/fr/LC_MESSAGES/djangojs.po:24]: --no-location

**Pull request merged**